src/ray/util/tests/pipe_logger_test.cc

```diff
@@ -224,7 +224,7 @@ TEST(PipeLoggerCompatTest, CompatibilityTest) {
     stream_redirection_handle.Close();
 
     const std::string stdout_content = testing::internal::GetCapturedStdout();
-    EXPECT_EQ(stdout_content, kContent);
+    EXPECT_EQ(stdout_content, "hello\nworld\n");
```

Comment:
why changing this? it should just match kContent right

Reply:
No, in the pr we use spdlog for all sinks, which means newliner at the end for everything.

Reply:
Yes, but for this test, kContent already has the newliner at the end.

Reply:
OK, thats for code style consistency, we're using raw string to match elsewhere